papaparse.js

```diff
@@ -643,6 +644,20 @@ License: MIT
 			}
 
 			xhr.open(this._config.downloadRequestBody ? 'POST' : 'GET', this._input, !IS_WORKER);
+			if (this._config.download && this._config.download.timeout) {
```

Comment:
I do not think using a nested option is a good idea.

Better to name it downloadTimeout

Reply:
Done